tracer/test/snapshots/Security.AspNetCore5AsmDataSecurityEnabled.__test=blocking-ips-oneclick_url=_.verified.txt

```diff
@@ -6,6 +6,37 @@
     Resource: GET /home/index,
     Service: Samples.Security.AspNetCore5,
     Type: web,
+    Tags: {
```

Comment:
I assume all snapshot changes are linked to the various fixes you've implemtented. I suppose Robert who knows better than I do as validated them when reviewing. So :blindfold: :)